t/is-and-extract-ipv6-gua.pl

```diff
@@ -0,0 +1,580 @@
+use Test::More;
+use B qw(perlstring);
+
+SKIP: { eval { require Test::Warnings; } or skip($@, 1); }
+eval { require 'ddclient'; } or BAIL_OUT($@);
+
+
+my @valid_ipv6_not_gua = (
```

Comment:
There is too much redundancy with `t/is-and-extract-ipv6.pl`. I would drastically simplify by testing only:
  * the last address before each rejected block
  * the first address of each rejected block
  * the last address of each rejected block
  * the first address after each rejected block

Reply:
I will update the tests